src/components/miner/cards/EpochStatus.svelte

```diff
@@ -1,6 +1,7 @@
 <script lang="ts">
   import { _ } from "svelte-i18n";
   import { onMount } from "svelte";
+import { _ } from "svelte-i18n";
```

Comment:
Not sure how is line is added?

Reply:
this duplicated lines are breaking build